hsa-rocr/PKGBUILD

```diff
@@ -15,15 +15,21 @@ makedepends=('cmake' 'libelf' "hsakmt-roct")
 provides=("rocr-runtime=$pkgver")
 replaces=('rocr-runtime')
 conflicts=('rocr-runtime')
-source=("$url/archive/rocm-$pkgver.tar.gz")
-sha256sums=('fa2d2d1f8a61d8a6952d377cf288d78c61776c3c2a666f163cafc3aa19ab0b61')
+source=("$url/archive/rocm-$pkgver.tar.gz"
+    'remove-warnings.patch')
+sha256sums=('fa2d2d1f8a61d8a6952d377cf288d78c61776c3c2a666f163cafc3aa19ab0b61'
+    '008cd076e6d30215b3e645586ae999aaaadfd7606d0b1ab313daa78d791440c6')
 _dirname="$(basename "$url")-$(basename "${source[0]}" .tar.gz)"
 
+prepare() {
+    cd "$srcdir/$_dirname/src"
+    patch -p0 < "$srcdir/remove-warnings.patch"
```

Comment:
Other patches usually run `patch -Np1 -i <patch file>`, can you see if that works here too? I'd rather have the same behavior everywhere.